src/cf-typography/usage.md

```diff
@@ -153,207 +185,79 @@ you should use the `aside` element.
 </aside>
 ```
 
+## Headings
 
-## Micro copy
-
-The `.micro-copy` class is good for highlighting small pieces of text.
-
-### Standard micro copy
-
-<p class="a-micro-copy">
-    Lorem ipsum dolor sit amet
-</p>
-
-```
-<p class="a-micro-copy">
-    Lorem ipsum dolor sit amet
-</p>
-```
-
-### Large micro copy
-
-<p class="a-micro-copy__large">
-    Lorem ipsum dolor sit amet
-</p>
-
-```
-<p class="a-micro-copy__large">
-    Lorem ipsum dolor sit amet
-</p>
-```
-
-## Date
-
-### Default date
-
-<span class="a-date">
-    Nov 4, 2013
-</span>
-
-```
-<span class="a-date">
-    Nov 4, 2013
-</span>
-```
-
+### Padded heading
 
-## Headers
+<h2 class="a-heading a-heading__padded">
+    Heading
+</h2>
 
-### Category slug
 
-The category slug is used on top of a section.
-Since categories can be repetitive,
-we suggest using a `.u-visually-hidden` element to add more context for screen readers.
+### Heading with icon
 
-#### Default category slug
+The heading with icon is typically used for listing categories in a meta header.
+Since categories can be repetitive, we suggest placing a label with `.u-visually-hidden`
+prior to the headings to add more context for screen readers (see Meta Header below).
 
-<a href="#" class="m-category-slug">
-    <span class="m-category-slug_icon cf-icon cf-icon-dialogue"></span>
-    <span class="u-visually-hidden">Category:</span>
```

Comment:
Sorry I wasn't clear but yes even if there was only one, it should still be included in the meta header. The singular/plural implementation can be left to the template generating it. I updated the name to header with icon because I believe it follows our atomic design goals and makes it a reusable pattern outside of categories. What example do you suggest instead of an existing category?